internal/utils/convert.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"strconv"
+)
+
+func ToBool(value interface{}) bool {
```

Comment:
I couldn't find a place for this kind of functions, but I could move all of the logic from the other getters as well to keep things clean. Just wanted to confirm this if this place is ok for them.